airbyte-integrations/connectors/source-shopify/unit_tests/integration/test_bulk_stream.py

```diff
@@ -38,7 +39,7 @@
 _JOB_START_DATE = datetime.fromisoformat("2024-05-05T00:00:00+00:00")
 _JOB_END_DATE = _JOB_START_DATE + timedelta(hours=2, minutes=24)
 
-_URL_GRAPHQL = f"https://{_SHOP_NAME}.myshopify.com/admin/api/2024-04/graphql.json"
+_URL_GRAPHQL = f"https://{_SHOP_NAME}.myshopify.com/admin/api/{ShopifyStream.api_version}/graphql.json"
```

Comment:
For the tests, we should be setting the static value here, because if we suddently change the version the test would be passed. We don't want this. This is a safaty net, basically, from dummy changes.